ui/pages/onboarding-flow/welcome/welcome.js

```diff
@@ -349,11 +348,13 @@ export default function OnboardingWelcome() {
 
         if (loginOption === LOGIN_OPTION.NEW) {
           await onSocialLoginCreateClick(loginType);
-          // if firefox, set isSocialLoginFlowEnabledForMetrics to false, otherwise set to true
-          dispatch(setIsSocialLoginFlowEnabledForMetrics(!isFireFox));
         } else if (loginOption === LOGIN_OPTION.EXISTING) {
           await onSocialLoginImportClick(loginType);
-          dispatch(setIsSocialLoginFlowEnabledForMetrics(false));
+        }
+
+        if (!isFireFox) {
+          // sent queued events
+          dispatch(setParticipateInMetaMetrics(true));
```

Comment:
so, on rehydration of social login on a new device, we make metametrics mandatory irrespective of their previous choice here?